examples/volume_plane_rendering.py

```diff
@@ -0,0 +1,230 @@
+
+        self.orientation_controls = QGroupBox('plane orientation controls')
+        orientation_controls_layout = QVBoxLayout(self.orientation_controls)
+
+        self.normal_rotate_0_positive_button = QPushButton('R(0)+', self)
+        self.normal_rotate_0_negative_button = QPushButton('R(0)-', self)
+        self.normal_rotate_1_positive_button = QPushButton('R(1)+', self)
+        self.normal_rotate_1_negative_button = QPushButton('R(1)-', self)
+        self.normal_rotate_2_positive_button = QPushButton('R(2)+', self)
+        self.normal_rotate_2_negative_button = QPushButton('R(2)-', self)
+
+        orientation_controls_layout.addWidget(self.normal_rotate_0_positive_button)
+        orientation_controls_layout.addWidget(self.normal_rotate_0_negative_button)
+        orientation_controls_layout.addWidget(self.normal_rotate_1_positive_button)
+        orientation_controls_layout.addWidget(self.normal_rotate_1_negative_button)
+        orientation_controls_layout.addWidget(self.normal_rotate_2_positive_button)
+        orientation_controls_layout.addWidget(self.normal_rotate_2_negative_button)
+
+        self.thickness = QGroupBox('thickness')
+        thickness_layout = QHBoxLayout(self.thickness)
+        self.thickness_spinbox = QLabeledDoubleSlider(Qt.Horizontal, self)
+        thickness_layout.addWidget(self.thickness_spinbox)
+        self.thickness_spinbox.setMinimum(1.0)
+        self.thickness_spinbox.setMaximum(64)
+        self.thickness_spinbox.setValue(10)
+
+        self.fun_buttons = QGroupBox('fun buttons')
+        fun_layout = QVBoxLayout(self.fun_buttons)
+
+        self.shift_normal_positive_button = QPushButton('shift along normal (+)')
+        self.shift_normal_negative_button = QPushButton('shift along normal (-)')
+        self.random_normal_button = QPushButton('random orientation', self)
+        self.random_position_button = QPushButton('random position', self)
+
+        fun_layout.addWidget(self.random_normal_button)
+        fun_layout.addWidget(self.random_position_button)
+        fun_layout.addWidget(self.shift_normal_positive_button)
+        fun_layout.addWidget(self.shift_normal_negative_button)
+
+        master_layout.addWidget(self.position_sliders)
+        master_layout.addWidget(self.orientation_presets)
+        master_layout.addWidget(self.orientation_controls)
+        master_layout.addWidget(self.thickness)
+        master_layout.addWidget(self.fun_buttons)
+
+    @property
+    def plane_position(self):
+        return self.position_0.value(), self.position_1.value(), self.position_2.value()
+
+
+def update_plane_position(plane_position):
+    viewer.layers['plane'].plane.position = plane_position
+    viewer.layers['plane point'].data = np.array(plane_position).reshape((-1, 3))
+
+    plane_normal_layer.data[:, 0, :] = plane_position
+    plane_normal_layer.data = plane_normal_layer.data
+
+
+def update_plane_normal(plane_normal):
```

Comment:
As soon as i click any of the rotation control buttons the plane normal disappears for me. It might be better to use a Line in the Shapes layer rather than a single vector in the vectors layer as i think it will render better in 3d too

Reply:
example is now much simpler, no more rotation control